src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/facebook/FacebookSettingsPageObject.java

```diff
@@ -64,14 +66,19 @@ public void removeAppIfPresent() {
           }
         }
       }
+      PageObjectLogging.log("removeApp", "Wikia App removed", true);
     }
-    PageObjectLogging.log("removeApp", "Wikia App removed", true);
   }
 
   /**
    * This method verifies if App is present on facebook apps list. It searches for Wikia string.
    */
   private boolean isAppPresent() {
+    try {
+      Thread.sleep(3000);
```

Comment:
The facebook/settings applications page takes a few seconds to load the Apps list -- I cannot wait for a specific element, as I don't know ahead of time if it will be on the page for certain, so I'd rather wait a determined amount of time before checking. Waiting for the high level element ID list was causing a failure regularly and this helps minimize that failure case.  If you have other ideas I'm happy to hear them.